packages/dd-trace/test/appsec/iast/analyzers/hardcoded-secret-analyzer.spec.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+const os = require('os')
+
+const agent = require('../../../plugins/agent')
+const Config = require('../../../../src/config')
+
+const hardcodedSecretAnalyzer = require('../../../../src/appsec/iast/analyzers/hardcoded-secret-analyzer')
+const { suite } = require('./resources/hardcoded-secrets-suite.json')
+const iast = require('../../../../src/appsec/iast')
+
+describe('Hardcoded Secret Analyzer', () => {
+  describe('unit test', () => {
+    const relFile = path.join('path', 'to', 'file.js')
+    const file = path.join(process.cwd(), relFile)
+    const line = 42
+    const column = 3
+
+    let report
+    beforeEach(() => {
+      report = sinon.stub(hardcodedSecretAnalyzer, '_report')
+    })
+
+    afterEach(sinon.restore)
+
+    suite.forEach((testCase) => {
+      testCase.samples.forEach(sample => {
+        it(`should match rule ${testCase.id} with value ${sample}`, () => {
+          hardcodedSecretAnalyzer.analyze({
+            file,
+            literals: [{
+              value: sample,
+              locations: [{
+                line,
+                column
+              }]
+            }]
+          })
+
+          expect(report).to.be.calledOnceWithExactly({ file: relFile, line, column, data: testCase.id })
```

Comment:
`to.have.been.called` is used +109 times in the repo, `to.be.called` ~60. I guess that we should use `to.have.been.called`
```suggestion
          expect(report).to.have.been.calledOnceWithExactly({ file: relFile, line, column, data: testCase.id })
```

Reply:
same for `not`. 
`not.to.have.been.called` is used more.